packages/frontend/src/assets/logos/amms/panko.tsx

```diff
@@ -0,0 +1,74 @@
+export function PankoLogo(props: any) {
+    return (
+        <svg
+            width="172"
+            height="32"
+            viewBox="0 0 172 32"
+            color="text"
+            xmlns="http://www.w3.org/2000/svg"
+            {...props}
+        >
+            <g clip-path="url(#clip0_2569_5438)">
+                <path
+                    fill-rule="evenodd"
```

Comment:
`camelCase` here to avoid console errors.

Reply:
Addressed in c74f1b7b7df6c5b951d4257a61666354c716b54a